java-postgres-container/pom.xml

```diff
@@ -0,0 +1,114 @@
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.github.cafapi</groupId>
+        <artifactId>caf-common</artifactId>
+        <version>1.11.0-SNAPSHOT</version>
+    </parent>
+
+    <groupId>com.hpe.caf</groupId>
+    <artifactId>java-postgres-container</artifactId>
+    <packaging>pom</packaging>
+    <version>1.0.0-SNAPSHOT</version>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>io.fabric8</groupId>
+                <artifactId>docker-maven-plugin</artifactId>
+                <version>${fabric8.docker.maven.version}</version>
+                <executions>
+                    <execution>
+                        <id>build-container</id>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>build</goal>
+                        </goals>
+                        <configuration>
+                            <skip>false</skip>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <phase>deploy</phase>
+                        <goals>
+                            <goal>push</goal>
+                        </goals>
+                        <configuration>
+                            <filter>java-postgres</filter>
+                        </configuration>
+                    </execution>
+                </executions>
+                <configuration>
+                    <verbose>true</verbose>
+                    <autoPull>always</autoPull>
+                    <images>
+                        <image>
+                            <alias>java-postgres</alias>
+                            <name>cafinternal/java-postgres:${project.version}</name>
+                            <build>
+                                <from>postgres:9.4.10-alpine</from>
+                                <labels>
+                                    <Build.Number>${project.version}</Build.Number>
+                                    <Build.Date>${maven.build.timestamp}</Build.Date>
+                                    <Git.Branch>${git.branch}</Git.Branch>
+                                    <Git.Commit>${git.revision}</Git.Commit>
+                                </labels>
+                                <optimise>false</optimise>
+                                <runCmds>
+                                    <runCmd>apk --update add openjdk8-jre</runCmd>
+                                </runCmds>
+                            </build>
+                            <run>
```

Comment:
I think the run section is beneficial even without integration tests;
1) Acts as an example for running the container conveniently located next to its build instructions.
2) Allows developer to start the container by running the docker:start maven goal against the pom if they want to verify something against the running container.
3) Provides a basic sanity check that the container starts up as part of the project build.

This pom was lacking the start and stop execution definitions so 3 wasn't applicable. I've added them to the pom now. If these don't seem good enough reasons I don't mind getting rid of it.